src/i18n.js

```diff
@@ -1,19 +1,99 @@
-import messagesNL from './i18n/compiled/nl.json';
-import messagesEN from './i18n/compiled/en.json';
-import { get } from 'api';
+import React, { useState } from "react";
+import { useAsync } from "react-use";
+import { IntlProvider } from "react-intl";
+
+import App from "components/App";
+import Loader from "components/Loader";
+
+import { get, post } from "api";
+import { ConfigContext, FormioTranslations } from "Context";
+
+import messagesNL from "./i18n/compiled/nl.json";
+import messagesEN from "./i18n/compiled/en.json";
 
 const loadLocaleData = (locale) => {
-    switch (locale) {
-        case 'nl':
-            return messagesNL;
-        default:
-            return messagesEN;
-    }
+  switch (locale) {
+    case "nl":
```

Comment:
please keep the code style consistent with existing code style in the file - we'll implement prettier for the SDK at some point, but javascript strings are generally single quote, while JSX props use double quotes.

These sort of formatting changes really clutter the diff and make reviewing harder.